### ISSUE TITLE
Cannot include repeat references to figure

### ISSUE THREAD

**Issue by MilesCranmer** [showyourwork/showyourwork#135]
I have a figure that I am showing multiple times in the same document (in my case, it is a multi-page PDF - each include is a different page, using `includegraphics[page=2]{…}`). 

This doesn’t seem to work right now. I see the following issue:
```
Rules syw__fig2 and syw__fig1 are ambiguous for the file src/tex/figures/pysr_diagram_v6.pdf
```

**Comment by MilesCranmer** [comment]
Looks like there are some options to fix this here: https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html?highlight=ruleorder#handling-ambiguous-rules

**Comment by MilesCranmer** [comment]
I was able to fix this locally by writing:

```
ruleorder: syw__fig1 > syw__fig2
```
in the Snakefile. Maybe this could be done automatically?

**Comment by rodluger** [comment]
Ah, interesting. Never thought of this failure mode!

Yes, it shouldn't be hard to fix this internally, since those two rules are identical -- they both run the same script to produce the same output. Should be easy to identify redundant rules internally and bypass this. Will work on a fix.

**Comment by rodluger** [comment]
@MilesCranmer This should be working on `main`, to be released in `0.3.1` in the next week. Feel free to re-open if you're still running into issues.
